runtime/lua/vim/lsp/buf.lua

```diff
@@ -150,6 +150,31 @@ local function select_client(method, on_choice)
   end
 end
 
+---@private
+---@return table {start={row, col}, end={row, col}} using (1, 0) indexing
+local function range_from_selection()
```

Comment:
Would you be interested in taking over https://github.com/neovim/neovim/pull/13896 ? Meanwhile should there be a comment here like:

    -- TODO: use `vim.region()` instead https://github.com/neovim/neovim/pull/13896

Reply:
Added the TODO. Currently I don't plan on taking over that PR, but that might change if I want this functionality in one of my plugins.